src/test/java/kostka/moviecatalog/service/RatingServiceIntegrationTest.java

```diff
@@ -0,0 +1,70 @@
+package kostka.moviecatalog.service;
+
+import kostka.moviecatalog.MovieCatalogApplication;
+import kostka.moviecatalog.dto.MovieDto;
+import kostka.moviecatalog.dto.RatingDto;
+import kostka.moviecatalog.entity.Movie;
+import kostka.moviecatalog.exception.InvalidDtoException;
+import kostka.moviecatalog.exception.MovieNotFoundException;
+import kostka.moviecatalog.repository.MovieElasticSearchRepository;
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.transaction.annotation.Transactional;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = MovieCatalogApplication.class)
+@Transactional
```

Comment:
Jestli to nejede, tak bych zkusil prihodit:
```@PropertySource("application.properties")``` - ale to by melo byt defaultne, tak uplne nevim. Zkusim to zitra zbuildit a pomoct ;)

Reply:
Hmm to je asi blbost. Mozna zkus jinou variantu z toho co sem ti posilal..

Reply:
Nejedou testy? Možná jiný runner? SpringJUnit4ClassRunner.class?